### DISCUSSION TITLE
Variable APPNAME n'apparait plus sur la première page.

### DISCUSSION THREAD

**Discussion by Nebrium** [dbarzin/mercator#890] [General]
Bonjour,
Dans l'ancienne version de mercator j'avais réglé la variable APPNAME du fichier .env. Celle-ci s'affichait sur l'écran de connexion.
J'ai fait la mise à jour la semaine dernière, et maintenant je n'ai plus le nom qui s'affiche en haut du formulaire de connexion.
Serait-il possible de le remettre ? J'ai 5 instances de mercator, et cela m'aide bien ;-)
Merci par avance !

**Comment by dbarzin** [comment]
OK, c'est à nouveau présent.

**Reply by Nebrium** [reply]
Merci !
